src/app/shared/contextHelper.ts

```diff
@@ -1,13 +1,12 @@
-import CONTEXTS from "../../constants/contexts";
-import PHONE_TYPES from "../../constants/phone-types";
+import { PHONE_TYPES } from "../../constants/device-constants";
 
 export const parseContextAsPhoneType = (context?: string): string => {
-  switch (context) {
-    case CONTEXTS.IPHONE:
-      return PHONE_TYPES.IPHONE;
-    case CONTEXTS.ANDROID:
-      return PHONE_TYPES.ANDROID;
-    default:
-      throw new Error(`Context cannot be parsed as a phone type: ${context}`);
+  if (
+    !context ||
+    !([PHONE_TYPES.IPHONE, PHONE_TYPES.ANDROID] as string[]).includes(context)
+  ) {
+    throw new Error(`Context cannot be parsed as a phone type: ${context}`);
   }
+
+  return context;
```

Comment:
If we're doing this, I wonder if it would have been better to switch this to `validatePhoneType` or something (no need to return the input)?

Doesn't really matter though.